scripts/protobuf_codegen.sh

```diff
@@ -14,13 +14,6 @@ RELAYER_PATH=$(
 
 source $RELAYER_PATH/scripts/versions.sh
 
-## ensure the correct version of "buf" is installed
-BUF_VERSION='1.36.0'
-if [[ $(buf --version | cut -f2 -d' ') != "${BUF_VERSION}" ]]; then
-  echo "could not find buf ${BUF_VERSION}, is it installed + in PATH?"
-  exit 255
-fi
```

Comment:
We should still keep this no? Otherwise calling this script on its own has no guarantee that the code is generated using the expected version.

Reply:
I've just added an install step here as well, which is basically a no-op if it's already there.